rest/http_api.md

```diff
@@ -616,7 +624,20 @@ Request body:
 }
 ```
 
-If omitted interface defaults to `custom`.
+Request body allowing to execute command on some selected node instances:
+
+```json
+{
+    "node": "NodeName",
+    "name": "Custom_Command_Name",
+    "interface": "fully.qualified.interface.name",
+    "instances": "[ "0", "1" ]",
```

Comment:
You should use single quotes or escape double quotes inside first ones as "[ \"0\", \"1\" ]"

Reply:
Fixed to 
"instances": [ "0", "1" ]